core/storage/user/gae_models.py

```diff
@@ -57,8 +57,7 @@ class UserSettingsModel(base_models.BaseModel):
     # User uploaded profile picture as a dataURI string. May be None.
     profile_picture_data_url = ndb.TextProperty(default=None, indexed=False)
     # The preferred dashboard of the user.
-    default_dashboard = ndb.StringProperty(
-        default=constants.DASHBOARD_TYPE_LEARNER, indexed=False)
+    default_dashboard = ndb.StringProperty(indexed=False)
```

Comment:
This still needs to default to something for new users, right? Probably should keep the default here...